src/dotnet/UI.Blazor/Components/ContextMenu/BlazorContextMenu/Components/ContextMenuBase.cs

```diff
@@ -0,0 +1,263 @@
+                (CssClass ?? template.MenuCssClass));
+        }
+    }
+
+    protected Animation GetActiveAnimation()
+    {
+        var animation = CascadingAnimation;
+        if (Animation != null)
+            animation = Animation;
+        if (animation == null)
+            animation = Settings.GetTemplate(GetActiveTemplate()).Animation;
+        return animation.Value;
+    }
+
+    internal string GetActiveTemplate()
+    {
+        var templateName = CascadingTemplate;
+        if (Template != null)
+            templateName = Template;
+        if (templateName == null)
+            templateName = BlazorContextMenuSettings.DefaultTemplateName;
+        return templateName;
+    }
+
+    protected string DisplayClassCalc {
+        get {
+            var template = Settings.GetTemplate(GetActiveTemplate());
+            var (showingAnimationClass, hiddenAnimationClass) = GetAnimationClasses(GetActiveAnimation());
+            return IsShowing ?
+                Helpers.AppendCssClasses(OverrideDefaultShownCssClass ?? template.DefaultCssOverrides.MenuShownCssClass,
+                    showingAnimationClass,
+                    ShownCssClass ?? Settings.GetTemplate(GetActiveTemplate()).MenuShownCssClass) :
+                Helpers.AppendCssClasses(OverrideDefaultHiddenCssClass ?? template.DefaultCssOverrides.MenuHiddenCssClass,
+                    hiddenAnimationClass,
+                    HiddenCssClass ?? Settings.GetTemplate(GetActiveTemplate()).MenuHiddenCssClass);
+        }
+    }
+    protected string ListClassCalc {
+        get {
+            var template = Settings.GetTemplate(GetActiveTemplate());
+            return Helpers.AppendCssClasses((OverrideDefaultListCssClass ?? template.DefaultCssOverrides.MenuListCssClass),
+                (ListCssClass ?? Settings.GetTemplate(GetActiveTemplate()).MenuListCssClass));
+        }
+    }
+
+    protected (string showingClass, string hiddenClass) GetAnimationClasses(Animation animation)
+    {
+        switch (animation)
+        {
+        case BlazorContextMenu.Animation.None:
+            return ("", "");
+        case BlazorContextMenu.Animation.FadeIn:
+            return ("blazor-context-menu__animations--fadeIn-shown", "blazor-context-menu__animations--fadeIn");
+        case BlazorContextMenu.Animation.Grow:
+            return ("blazor-context-menu__animations--grow-shown", "blazor-context-menu__animations--grow");
+        case BlazorContextMenu.Animation.Slide:
+            return ("blazor-context-menu__animations--slide-shown", "blazor-context-menu__animations--slide");
+        case BlazorContextMenu.Animation.Zoom:
+            return ("blazor-context-menu__animations--zoom-shown", "blazor-context-menu__animations--zoom");
+        default:
+            throw new Exception("Animation not supported");
+        }
+    }
+
+    protected override void OnInitialized()
+    {
+        base.OnInitialized();
```

Comment:
Check if it does something; if not, just remove the call.

Reply:
removed